common/display/displayqueue.cpp

```diff
@@ -111,10 +111,10 @@ bool DisplayQueue::Initialize(uint32_t width, uint32_t height, uint32_t pipe,
   drmModePropertyPtr broadcastrgb_props =
       drmModeGetProperty(gpu_fd_, broadcastrgb_id_);
 
-  if (!(broadcastrgb_props->flags & DRM_MODE_PROP_ENUM))
+  if (broadcastrgb_props && !(broadcastrgb_props->flags & DRM_MODE_PROP_ENUM))
```

Comment:
You can make this || condition and include check !broadcastrgb_propos this way we should just return false here, since it doesnt make sense to go any further in case its null